src/dapp-tests/integration/tests.sh

```diff
@@ -292,3 +292,73 @@ test_hexdata_3() {
   [[ $(seth --to-hexdata 0xCA:0xfe:0x) = "0xcafe" ]] || error
 }
 test_hexdata_3
+
+# SETH ENS TESTS
+# Tests for resolve-name and lookup-address may break in the future as they are dependent on mainnet state
+# Infura ID source: https://github.com/ethers-io/ethers.js/blob/0d40156fcba5be155aa5def71bcdb95b9c11d889/packages/providers/src.ts/infura-provider.ts#L17
+ETH_RPC_URL=https://mainnet.infura.io/v3/84842078b09946638c03157f83405213
```

Comment:
I'm a bit concerned about these mainnet tests that can easily break some day. Are there any ENS names which are guaranteed to be there forever? Using historical block isn't really an option either since it would require an archive node.

Reply:
Yea agreed, the current approach isn't ideal. There are some names registered for a really long time, but none that we can guarantee won't change their resolver configuration. I think most robust solution would be to register `seth-test.eth` on Rinkeby for 100 or so years, configure the resolver, then never  touch it again. I don't mind setting that up if that works for you

Reply:
That sounds like a good solution to me.

Reply:
Done in c69851381ec39537ff6a740ef8e70d02d72deab7